### PR TITLE
Done but may need some changes

### PR DESCRIPTION
**Pull request by nagy3n**
Resolving #5 

May need some UI changes

### CODE REVIEW

File: ma-copy-artist-album-to-clipboard.user.js [hunk 1]
```diff
@@ -41,15 +41,40 @@ const tmpl = `<div class="copy-artist-notification"></div>
...
+        GM_setClipboard(cpStrs + "\n");
+        popUp.innerText = `Copied all to clipboard`;
+        setTimeout(() => {
+            popUp.style.display = 'none';
+        }, 2000);
+        popUp.style.display = 'block';
+    });
+
+
+    const nameColumn = document.querySelector("#ui-tabs-4 > table > thead > tr > th.releaseCol");
```
Comment (Row) [inline ma-copy-artist-album-to-clipboard.user.js:74]:
```suggestion
    const nameColumn = document.querySelector("#ui-tabs-4 > table > thead > tr > th.releaseCol");
    nameColumn.style.position = 'relative'
```

File: ma-copy-artist-album-to-clipboard.user.js [hunk 2]
```diff
@@ -41,15 +41,40 @@ const tmpl = `<div class="copy-artist-notification"></div>
...
         insertPoint.style.position = 'relative';
         elements.forEach(elem => {
             link.parentNode.prepend(elem);
         });
     });
+    const cpStrs = cpArr.join("\n")
+    generateCopyAllButton(tmpl, cpStrs)
+}
+
+function generateCopyAllButton(html, cpStrs) {
```
Comment (Row) [inline ma-copy-artist-album-to-clipboard.user.js:59]: It would probably be good to reuse the generateTemplate function, but that can be fixed later.

**Review by Row — CHANGES_REQUESTED** [review]
Perhaps you could update the commit message to describe the change better.

File: ma-copy-artist-album-to-clipboard.user.js (hunk 2 above)
Comment (nagy3n) [reply ma-copy-artist-album-to-clipboard.user.js:59]: Yeah with some flags for sure

**Pull request closed without merging**